### PR TITLE
initial ProcessingVersion include

### LINKED ISSUE #5793: SubscriptionInformation not in workload cache
Also add ProcessingVersion as well

On @amaltaro suggestion:

neither of

https://cmsweb.cern.ch/couchdb/reqmgr_workload_cache/jbadillo_TOP-Summer12-00335_00209_v0__150406_103823_3394

  nor

https://cmsweb.cern.ch/reqmgr2/data/request?name=jbadillo_TOP-Summer12-00335_00209_v0__150406_103823_3394

  has

'SubscriptionInformation':
{'/TTJets_MSDecays_JpsiFilter_1785_TuneZ2star_8TeV-madgraph-tauola/Summer12-START53_V7C-v2/GEN-SIM':
{'NonCustodialSites': [], 'CustodialSubType': 'Move', 'CustodialSites':
[], 'Priority': 'Low', 'AutoApproveSites': [], 'NonCustodialSubType':
'Replica'}}

  present in :

https://cmsweb.cern.ch/reqmgr/reqMgr/request?requestName=jbadillo_TOP-Summer12-00335_00209_v0__150406_103823_3394

### PR DESCRIPTION
**Pull request by ticoann**
fixes #5793.  This can't be patched to agent until cmsweb is updated.
after all the agent are deployed with this included, map.js need to be cleaned up to accept only request body.
1. Fixes #5793 
2. Also adds all the TaskChain params in reqmgr_workload_cache db but only when reqmgr2 api used.

Still need to be tested. Don't merge.

### CODE REVIEW

**Comment by ticoann** [comment]
@amaltaro, Alan, please review.

File: src/couchapps/ReqMgr/updates/updaterequest.js [hunk 1]
```diff
@@ -29,28 +29,45 @@ function(doc, req) {
     // req.query is dictionary fields into the 
     // CMSCouch.Database.updateDocument() method, which is a dictionary
     var newValues = req.query;
```
Comment (amaltaro) [inline src/couchapps/ReqMgr/updates/updaterequest.js]: This line can be removed, new Values receives new content a few lines below.

**Comment by amaltaro** [comment]
Besides that short comment, patch looks good to me.

**Comment by ticoann** [comment]
Thanks Alan,  I tested in vm as well. merging it.

**Pull request merged**